components/centraldashboard/public/components/main-page.js

```diff
@@ -134,8 +133,9 @@ export class MainPage extends PolymerElement {
      * @param {MouseEvent} e
      */
     openInIframe(e) {
-        const url = new URL(e.currentTarget.href);
-        window.history.pushState({}, null, `_${url.pathname}`);
+        // e.currentTarget is an anchor element
+        const url = e.currentTarget.href.substr(e.currentTarget.origin.length);
```

Comment:
I think `slice` is the recommended js-onic way of substringing:
```suggestion
        const url = e.currentTarget.href.slice(e.currentTarget.origin.length);
```

Reply:
Changed.